pkg/prompts/prompts.go

```diff
@@ -37,7 +37,7 @@ type Prompter interface {
 	CaptureDuration(promptStr string) (time.Duration, error)
 	CaptureDate(promptStr string) (time.Time, error)
 	CaptureNodeID(promptStr string) (ids.NodeID, error)
-	CaptureWeight(promptStr string) (uint64, error)
+	CaptureWeight(promptStr string) (int64, error)
```

Comment:
why int ? negative values?

Reply:
I didn't realize I could use uint in the flag :man_facepalming: